src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx

```diff
@@ -104,6 +105,7 @@ export function ComboboxMulti<S>({
                         e?.stopPropagation();
                     },
                 })}
+                role="img"
```

Comment:
I think this should have `role="button"`

Reply:
:+1: